### ISSUE TITLE
Edit credit for Coding image for redesigned homepage

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3295]
### Overview
We need to edit a credit for a new image used in the redesigned version of the homepage so that the [Credits page](https://www.hackforla.org/credits/) is up to date.

### Details
From the [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724), here is a screenshot of the credit's information for reference:
  <details>
    <summary>Click here to see the screenshot of the Coding credit in Figma</summary>
    <img src="https://user-images.githubusercontent.com/31293603/175393444-ef5d9cdc-ca7d-4ee7-8779-eebcbc459c99.png" alt="">
  </details>

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch`. Please follow ["Step 1: Create your branch" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-1-create-your-branch) wiki article.
- [x] For `_data/internal/credits/engineering.yml`,
  - [x] Change the file name from `engineering.yml` to `coding.yml`
  - [x] Change `title: Engineering` to `title: Coding`
  - [x] Change 
    `title-link: https://thenounproject.com/ralfschmitzer/collection/gui-basics/?i=486344`
    to
    `title-link: https://thenounproject.com/search/?q=coding&i=486344`
  - [x] Change `content: icon` to `content-type: image`
  - [x] Change `used-in: Home` to `used-in: Homepage`
  - [x] Change `alt: 'Engineering Icon'` to `alt: ''`
  - [x] Remove `type: icon`
- [x] To update your topic branch, please follow ["Step 2: Update your branch with upstream changes" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-2-update-your-branch-with-upstream-changes) wiki article.
- [x] The pull request created from this issue should be merged into `feature-homepage-launch`. Please follow ["Step 3: Create a pull request" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-3-create-a-pull-request) wiki article
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3130 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3130 to the New Issue Approval column and remove the Dependency label
  
### Resources/Instructions
- This issue is tracked in the epic #3130.
- [Credits page](https://www.hackforla.org/credits/)
- [How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [Credits yml file template for Iconography and Imagery](https://github.com/hackforla/website/wiki/Credits-yml-file-template-for-Iconography-and-Imagery)
- [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724)

**Comment by JessicaLucindaCheng** [comment]
This issue would usually have the milestone 01. Compliance. However, it's specifically for the homepage launch, which has not happened yet.

**Comment by neevliberman** [comment]
Progress: Claimed issue, need more time to look at it: very busy
Blockers: None
Availability: 2 hours
ETA: 2022-08-20
